### PR TITLE
mount.py - lazy umount, trailing slash tolerance and missing dump/passno tolerance

### PR DESCRIPTION
**Pull request by zimo2001**
mount module:
1. Added feature - "lazy" umount (by combining "lazy=yes" with "state=umounted/absent")
2. set_mount function - Added ability to compare a given mount with a line in fstab that doesn't contain dump/passno
3. set_mount & unset_mount functions - mount names that have a trailing slash won't be treated as a different mount

### CODE REVIEW

File: system/mount.py [hunk 1]
```diff
@@ -124,6 +126,8 @@ def set_mount(module, **kwargs):
     origname =  args['name']
     # replace any space in mount name with '\040' to make it fstab compatible (man fstab)
     args['name'] = args['name'].replace(' ', r'\040')
+    # remove trailing slash in mount name (if exists)
+    args['name'] = re.sub('/$','',args['name'])
```
Comment (parberge) [inline system/mount.py]: Why not use rstrip instead which will remove the need of re module:
`args['name'] = args['name'].rstrip('/')`

Comment (zimo2001) [inline system/mount.py]: Hi @peerster!

Thanks for the feedback. You're absolutely right, rstrip is a better method.

But - I'm kinda new at github...
Should I edit the code or create a new commit and then a new pull request?

Comment (parberge) [inline system/mount.py]: I'm no expert either, but I think you can just commit new changes and this
PR will be updated with them. That said, I don't know if the project wants
to handle it differently.  Maybe there is some info in the contribution
info?

Comment (zimo2001) [inline system/mount.py]: thanks @peerster! commited...

**Comment by gregdek** [comment]
Thanks @zimo2001 for your pull request. This module is critical to Ansible's functionality, so it's owned by the Core team, and it can take a while for patches to be reviewed. Thanks for your patience here.

**Comment by gregdek** [comment]
Thanks @zimo2001 for this PR. Unfortunately, it is not mergeable in its current state due to merge conflicts. Please rebase your PR. When you are done, please comment with text 'ready_for_review' and we will put this PR back into review.

**Comment by gregdek** [comment]
@zimo2001 A friendly reminder: this pull request has been marked as needing your action, and it hasn't been updated in the last month. 

If you still believe that this PR applies, and you intend to address the issues with this PR, just let us know in the PR itself and we'll keep it open pending your changes.

If we receive no indication from you within a week, we will close this pull request.

**Comment by gregdek** [comment]
Thanks @zimo2001 for this PR. Unfortunately, it is not mergeable in its current state due to merge conflicts. Please rebase your PR. When you are done, please comment with text 'ready_for_review' and we will put this PR back into review.

[This message brought to you by your friendly Ansibull-bot.]

**Comment by gregdek** [comment]
@zimo2001 A friendly reminder: this pull request has been marked as needing your action. If you still believe that this PR applies, and you intend to address the issues with this PR, just let us know in the PR itself and we will keep it open pending your changes. When you do address the issues, please respond with ready_for_review in your comment, so that we can notify the maintainer.

[This message brought to you by your friendly Ansibull-bot.]

**Comment by gregdek** [comment]
@zimo2001 Another friendly reminder: this pull request has been marked as needing your action. If you still believe that this PR applies, and you intend to address the issues with this PR, just let us know in the PR itself and we will keep it open. If you have addressed the issues and believe it's ready for review, please comment with the text "ready_for_review". If we don't hear from you within another 14 days, we will close this pull request.

[This message brought to you by your friendly Ansibull-bot.]

**Comment by gregdek** [comment]
@zimo2001 A friendly reminder: this pull request has been marked as needing your action. If you still believe that this PR applies, and you intend to address the issues with this PR, just let us know in the PR itself and we will keep it open pending your changes. When you do address the issues, please respond with ready_for_review in your comment, so that we can notify the maintainer.

[This message brought to you by your friendly Ansibull-bot.]

**Comment by gregdek** [comment]
@zimo2001 Another friendly reminder: this pull request has been marked as needing your action. If you still believe that this PR applies, and you intend to address the issues with this PR, just let us know in the PR itself and we will keep it open. If you have addressed the issues and believe it's ready for review, please comment with the text "ready_for_review". If we don't hear from you within another 14 days, we will close this pull request.

[This message brought to you by your friendly Ansibull-bot.]

**Comment by gregdek** [comment]
@zimo2001 A friendly reminder: this pull request has been marked as needing your action. If you still believe that this PR applies, and you intend to address the issues with this PR, just let us know in the PR itself and we will keep it open pending your changes. When you do address the issues, please respond with ready_for_review in your comment, so that we can notify the maintainer.

[This message brought to you by your friendly Ansibull-bot.]

**Comment by gregdek** [comment]
@zimo2001 Another friendly reminder: this pull request has been marked as needing your action. If you still believe that this PR applies, and you intend to address the issues with this PR, just let us know in the PR itself and we will keep it open. If you have addressed the issues and believe it's ready for review, please comment with the text "ready_for_review". If we don't hear from you within another 14 days, we will close this pull request.

[This message brought to you by your friendly Ansibull-bot.]

**Comment by gregdek** [comment]
@zimo2001 A friendly reminder: this pull request has been marked as needing your action. If you still believe that this PR applies, and you intend to address the issues with this PR, just let us know in the PR itself and we will keep it open pending your changes. When you do address the issues, please respond with ready_for_review in your comment, so that we can notify the maintainer.

[This message brought to you by your friendly Ansibull-bot.]

**Comment by gregdek** [comment]
@zimo2001 Another friendly reminder: this pull request has been marked as needing your action. If you still believe that this PR applies, and you intend to address the issues with this PR, just let us know in the PR itself and we will keep it open. If you have addressed the issues and believe it's ready for review, please comment with the text "ready_for_review". If we don't hear from you within another 14 days, we will close this pull request.

[This message brought to you by your friendly Ansibull-bot.]

**Comment by gregdek** [comment]
@zimo2001 A friendly reminder: this pull request has been marked as needing your action. If you still believe that this PR applies, and you intend to address the issues with this PR, just let us know in the PR itself and we will keep it open pending your changes. When you do address the issues, please respond with ready_for_review in your comment, so that we can notify the maintainer.

[This message brought to you by your friendly Ansibull-bot.]

**Comment by gregdek** [comment]
@zimo2001 Another friendly reminder: this pull request has been marked as needing your action. If you still believe that this PR applies, and you intend to address the issues with this PR, just let us know in the PR itself and we will keep it open. If you have addressed the issues and believe it's ready for review, please comment with the text "ready_for_review". If we don't hear from you within another 14 days, we will close this pull request.

[This message brought to you by your friendly Ansibull-bot.]

**Comment by gregdek** [comment]
@zimo2001 A friendly reminder: this pull request has been marked as needing your action. If you still believe that this PR applies, and you intend to address the issues with this PR, just let us know in the PR itself and we will keep it open pending your changes. When you do address the issues, please respond with ready_for_review in your comment, so that we can notify the maintainer.

[This message brought to you by your friendly Ansibull-bot.]

**Comment by gregdek** [comment]
@zimo2001 Another friendly reminder: this pull request has been marked as needing your action. If you still believe that this PR applies, and you intend to address the issues with this PR, just let us know in the PR itself and we will keep it open. If you have addressed the issues and believe it's ready for review, please comment with the text "ready_for_review". If we don't hear from you within another 14 days, we will close this pull request.

[This message brought to you by your friendly Ansibull-bot.]

**Comment by gregdek** [comment]
@zimo2001 A friendly reminder: this pull request has been marked as needing your action. If you still believe that this PR applies, and you intend to address the issues with this PR, just let us know in the PR itself and we will keep it open pending your changes. When you do address the issues, please respond with ready_for_review in your comment, so that we can notify the maintainer.

[This message brought to you by your friendly Ansibull-bot.]

**Comment by gregdek** [comment]
@zimo2001 Another friendly reminder: this pull request has been marked as needing your action. If you still believe that this PR applies, and you intend to address the issues with this PR, just let us know in the PR itself and we will keep it open. If you have addressed the issues and believe it's ready for review, please comment with the text "ready_for_review". If we don't hear from you within another 14 days, we will close this pull request.

[This message brought to you by your friendly Ansibull-bot.]

**Pull request closed without merging**